walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,58 @@
 
 package com.google.sps;
 
+import java.util.Comparator;
 import java.util.Collection;
+import static java.util.Collections.disjoint;
+import static java.util.Collections.sort;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
You might want a Set instead of a Collection so it gets rid of duplicates.  (I assume you don't care about order.)

Reply:
I want a collection because I do order the items in the collection.